dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -36,19 +37,33 @@
 
   private static final Logger logger = Logger.getLogger(DynatraceMetricExporter.class.getName());
 
-  private DynatraceMetricExporter(URL url, String apiToken, Boolean enrichWithOneAgentMetaData) {
+  private DynatraceMetricExporter(
+      URL url,
+      String apiToken,
+      String prefix,
+      Labels defaultDimensions,
+      Boolean enrichWithOneAgentMetaData) {
     this.url = url;
     this.apiToken = apiToken;
 
-    Collection<AbstractMap.SimpleEntry<String, String>> localTags = new ArrayList<>();
+    Collection<AbstractMap.SimpleEntry<String, String>> defaultAndOneAgentDimensions =
+        new ArrayList<>();
 
     if (enrichWithOneAgentMetaData) {
       OneAgentMetadataEnricher enricher = new OneAgentMetadataEnricher(logger);
-      localTags.addAll(enricher.getDimensionsFromOneAgentMetadata());
+      defaultAndOneAgentDimensions.addAll(enricher.getDimensionsFromOneAgentMetadata());
     }
 
+    if (defaultDimensions != null) {
+      defaultDimensions.forEach(
+          (String k, String v) -> {
+            defaultAndOneAgentDimensions.add(new AbstractMap.SimpleEntry<>(k, v));
+          });
+    }
+
+    MetricAdapter.getInstance().setPrefix(prefix);
     // add the tags to the MetricAdapter.
-    MetricAdapter.getInstance().setTags(localTags);
+    MetricAdapter.getInstance().setTags(defaultAndOneAgentDimensions);
```

Comment:
This should be renamed to `setDefaultDimensions(...)` when adding the library to completely remove the "tags" wording.